### PR TITLE
Make berry Hints useful

### PR DESCRIPTION
**Pull request by DraKay**
Hi, is me again, your local suerhero.... **The FarmerWatcher**, my mission, make everyone understand the farm, living in the shadows, always on duty, each second of my life dedicated to that 25 little plots.

Today mission is make possible to mutate any berry without need the wiki (yes it goes against my wiki contributor secret identity...) and how is that possible? Making berry hints not confusing any more with the following changes:

1. Add a table on help tab of berry modal making equivalency between the descriptors that the grumpy grandpa (berry Kanto master) uses and flavour points so is posible to know wich flavour thresholds should be meet (idea of lucho) 
2. Specify max amount of other flavours in hints for flavour mutations (English vocabulary courtesy of Lucho) 
3. Change lum berry hint saying 8 more basics berries (idea of Darkmazer) 
4. Change pinkan berry to specify it needs 7 pink berries and put that they are in pinkan mountain (idea of darkmazer) 
5. Add the amount of berries needed for legendary as farm term is wide used but with not congruent meanings so was confusing
6. Change petaya hint to say something...

### CODE REVIEW

**Comment by CypherX** [comment]
Could use some grammar and spelling improvements.

Hint:

> I've heard that a special Berry can appear if its surroundings get too ..., without a minimal amount of other flavors!

Improved:

> I've heard that a special Berry can appear if its surroundings get too ... while keeping other flavors to a minimum!

Hint:

> I've heard that a special Berry can appear if its surroundings match its flavor profile! If I recall, it tasted a little spicy, a little sweet, a little sour and nothing more at the same time.

Improved:

> I've heard that a special Berry can appear if its surroundings match its flavor profile! If I recall, it tasted a little spicy, a little sweet, a little sour, and nothing else!

Hint:

> I've heard that a special Berry can appear if its surroundings match its flavor profile! If I recall, it tasted fairly spicy, dry, bitter and nothing more at the same time.

Improved:

> I've heard that a special Berry can appear if its surroundings match its flavor profile! If I recall, it tasted fairly spicy, dry, and bitter with no hints of any other flavor!

Hint:

> I've heard that a ... berry will change if its surroundings get extremely ...,bbut without a high amount of other flavors!

Improved:

> I've heard that a ... berry will change if its surroundings get extremely ... while keeping other flavors to a minimum!

Hint:

> I've heard that there's a special Pink Berry that only appears when surrounded by a seven different types of Pink Berries natives to Pinkan Mountains!

Improved:

> I've heard that there's a special Pink Berry that only appears when surrounded by the seven different types of Pink Berries native to Pinkan Mountains!

Hint:

> I've heard that a special Berry can appear after being totally surrounded by Rindo Berries!

Improved:

> I've heard that a special Berry can appear after being completely surrounded by Rindo Berries!

Hint:

> Legends tell of a mysterious Berry that only appears in a field with berries from Occa to Roseli

Improved:

> Legends tell of a mysterious Berry that only appears in a field with a wide variety of berries from Occa to Roseli.

**Comment by CypherX** [comment]
Some more:

Text:

> <b>Evolution mutations:</b> They usually require a berry changing into other and are related with flavor profiles or use of objects. <i>This mutations can start to happen at Taller stage.</i>

Improved:

> <b>Evolution mutations:</b> The result of a berry changing into another due to its surrounding flavor profile or use of specific items. <i>These mutations will start to occur at the Taller stage.</i>

Text (I know this one wasn't changed in this PR but might as well fix it):

> <b>Common mutations:</b> They requires specific berries to happen. <i>This mutations only happens at Ripe stage.

Improved:

> <b>Common mutations:</b> Growing specific berries together can cause a new berry to sprout nearby. <i>These mutations only happen at the Ripe stage.</i>

**Comment by EatPant2nd** [comment]
While we call it "Ripe" stage, the game calls it "Berry" stage, so I would use that word instead.

**Comment by DraKay** [comment]
Changed both, time and pant, as I said in one of the commits:

I haven't touch the ones that says without a high amount as using the same expresión that figy defeat the purpose of this pr, one is <=5 and other <=80 and can't be expressed same way. Also didn't touch petaya yet because there is discusión of how it should be expressed and might change

Feel free to tell grammar/vocabulary mistakes or help to improve petaya hint

File: src/scripts/farming/Farming.ts [hunk 1]
```diff
@@ -645,7 +645,7 @@ class Farming implements Feature {
                 BerryType.Magost,
                 BerryType.Watmel,
             ], {
-                hint: 'I\'ve heard that there\'s a special Pink Berry that only appears when surrounded by a bunch of different types of Pink Berries!',
+                hint: 'I\'ve heard that there\'s a special Pink Berry that only appears when surrounded by the seven different types of Pink Berries natives to Pinkan Mountains!',
```
Comment (CypherX) [inline src/scripts/farming/Farming.ts:648]: `natives` should be `native`

File: src/scripts/farming/Farming.ts [hunk 2]
```diff
@@ -608,30 +608,30 @@ class Farming implements Feature {
...
         this.mutations.push(new EvolveNearFlavorMutation(.0002, BerryType.Durin, BerryType.Rabuta,
             [[0, 80], [0, 80], [0, 80], [130, 160], [0, 80]], {
-                hint: 'I\'ve heard that a Rabuta berry will change if its surroundings get extremely bitter!',
+                hint: 'I\'ve heard that a Rabuta berry will change if its surroundings get extremely bitter, but without a high amount of other flavors!!',
             }));
         // Belue
         this.mutations.push(new EvolveNearFlavorMutation(.0002, BerryType.Belue, BerryType.Nomel,
             [[0, 80], [0, 80], [0, 80], [0, 80], [130, 160]], {
-                hint: 'I\'ve heard that a Nomel berry will change if its surroundings get extremely sour!',
+                hint: 'I\'ve heard that a Nomel berry will change if its surroundings get extremely sour,bbut without a high amount of other flavors!!',
```
Comment (CypherX) [inline src/scripts/farming/Farming.ts:634]: typo `bbut`

File: src/scripts/farming/Farming.ts [hunk 3]
```diff
@@ -727,7 +727,7 @@ class Farming implements Feature {
         let berryReqs = {};
         berryReqs[BerryType.Rindo] = 8;
         this.mutations.push(new GrowNearBerryStrictMutation(.0001, BerryType.Tanga, berryReqs, {
-            hint: 'I\'ve heard that a special Berry can appear after being surrounded by Rindo Berries!',
+            hint: 'I\'ve heard that a special Berry can appear after being completly surrounded by Rindo Berries!',
```
Comment (CypherX) [inline src/scripts/farming/Farming.ts:730]: `completly` should be `completely`

**Comment by DraKay** [comment]
Changed 👍

**Pull request closed without merging**